### DISCUSSION TITLE
Oops，这里是 404 页面。

### DISCUSSION THREAD

**Discussion by NickMao05** [spencerwooo/onedrive-vercel-index#929] [Q&A] [answered]
api确定没有问题，权限也没有问题，site设置也没有问题
清除redis 缓存后重新认证，但问题依旧
![227765713-7649a672-b640-469a-b5f0-d1f5f3a349be](https://user-images.githubusercontent.com/69585388/227766173-3f142d14-9a5b-4ae0-98ac-cb53c79aae9a.png)
![227765718-e7724deb-8a75-4c8c-a8fe-7af39a7fe226](https://user-images.githubusercontent.com/69585388/227766178-19b5544f-a191-4205-8405-2368abf75dc4.png)
![227765722-6653b75a-9e78-4eec-b206-31d8a7a1ac1c](https://user-images.githubusercontent.com/69585388/227766182-d53b36a5-6925-4567-8ed3-955bee3c95d3.png)

{"error":{"error":{"code":"itemNotFound","message":"The resource could not be found.","innerError":{"date":"2023-03-26T09:02:50","request-id":"2daa98bf-0d7b-44aa-a19e-0bee749162b8","client-request-id":"2daa98bf-0d7b-44aa-a19e-0bee749162b8"}}}}

**Comment by NickMao05** [comment]
![image](https://user-images.githubusercontent.com/69585388/227775546-ae85da4b-8ea9-4c06-a7fa-6272e8d80c3f.png)
[https://github.com/spencerwooo/onedrive-cf-index](url)
但是作者以前cloudflare worker的项目却可以工作🤔

**Reply by NickMao05** [reply]
会不会是以前onedrive-cf-index与现在此项目请求graph api的方式不一样导致的问题？
@spencerwooo

**Comment by nimmadev** [comment]
Same error

**Reply by alexlamazing** [reply]
今天新安裝的，也是 404
嘗試了 https://github.com/spencerwooo/onedrive-vercel-index/discussions/886 提到的方法，清除 redis 後重新驗證也是不行

**Reply by NickMao05** [reply]
#886 的404问题可能和该问题不一样

**Comment by spencerwooo** [comment, marked as answer]
是资源目录没有配置好。如果需要暴露根目录，需要修改配置文件 `site.config.js`：

```diff
-baseDirectory: '/Public',
+baseDirectory: '/',
```
